<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="83" started-at="2018-04-12T06:12:51Z" finished-at="2018-04-12T06:12:51Z">
    <groups>
      <group name="unit">
        <method signature="SampleTest.first()[pri:0, instance:samples.SampleTest@1b0375b3]" name="first" class="samples.SampleTest"/>
      </group> <!-- unit -->
      <group name="web">
        <method signature="SampleTest.second()[pri:0, instance:samples.SampleTest@1b0375b3]" name="second" class="samples.SampleTest"/>
      </group> <!-- web -->
    </groups>
    <test name="test1" duration-ms="45" started-at="2018-04-12T06:12:51Z" finished-at="2018-04-12T06:12:51Z">
      <class name="samples.SampleTest">
        <test-method status="PASS" signature="first()[pri:0, instance:samples.SampleTest@1b0375b3]" name="first" duration-ms="11" started-at="2018-04-12T00:12:51Z" finished-at="2018-04-12T00:12:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- first -->
        <test-method status="FAIL" signature="second()[pri:0, instance:samples.SampleTest@1b0375b3]" name="second" duration-ms="3" started-at="2018-04-12T00:12:51Z" finished-at="2018-04-12T00:12:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at samples.SampleTest.second(SampleTest.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:379)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:340)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:413)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- second -->
      </class> <!-- samples.SampleTest -->
    </test> <!-- test1 -->
    <test name="test2" duration-ms="0" started-at="2018-04-12T06:12:51Z" finished-at="2018-04-12T06:12:51Z">
    </test> <!-- test2 -->
  </suite> <!-- Suite1 -->
</testng-results>
